### DISCUSSION TITLE
add custom static files to /public folder

### DISCUSSION THREAD

**Discussion by matikepa** [chrede88/qubt#173] [Q&A] [answered]
I'm hosting my site on GitHub Pages. To set up a custom domain, I need to create a CNAME file directly in the repository. 

Is there an option in your template to place any file in a specific location with the desired content?

**Comment by chrede88** [comment, marked as answer]
I don't think I completely understand your issue here. Can't you just add a CNAME file to your repo?

**Reply by matikepa** [reply]
should I add it to any particular folder like `./static`?

**Reply by matikepa** [reply]
okay nvm , seems to work that way. Sorry for trouble

**Reply by chrede88** [reply]
This is a GitHub thing. Shouldn't have anything to do with Hugo or the theme.
